resolwe_bio/descriptors/reads_template.yml

```diff
@@ -0,0 +1,13 @@
+- name: reads_annotation
+  label: Reads annotation
+  description: Annotation of raw sequencing data
+  var:
```

Comment:
@mstajdohar: we map `var` section of the file to the `schema` field in model. Maybe we should unify the naming?